src/test/java/com/github/javafaker/StarTrekTest.java

```diff
@@ -26,4 +27,7 @@ public void specie() {
     public void villain() {
         assertThat(faker.starTrek().villain(), matchesRegularExpression("^(\\w+'?\\.?\\s?)+$"));
     }
+
+    @Test
+    public void klingon() { assertFalse(faker.starTrek().klingon().isEmpty()); }
```

Comment:
Can you use the hamcrest matches please so it is consistent with the rest of the test?